mod-circulation/src/main/resources/vega/mod-circulation/features/tlr-requests.feature

```diff
@@ -163,3 +163,64 @@ Feature: Title level request tests
     And match $.requests[2].instanceId == extInstanceId
     And match $.requests[2].position == 3
 
+  Scenario: TLR recall should not prohibit checkout of another item of the same instance
+    * def extMaterialTypeId = call uuid1
+    * def extMaterialTypeName = 'book'
+    * def extUserId1 = call uuid1
+    * def extUserId2 = call uuid1
+    * def extUserId3 = call uuid1
+    * def extUserBarcode1 = 'FAT-6950UBC-1'
+    * def extUserBarcode2 = 'FAT-6950UBC-2'
+    * def extUserBarcode3 = 'FAT-6950UBC-3'
+    * def extItemId1 = call uuid1
+    * def extItemId2 = call uuid1
+    * def extItemBarcode1 = 'FAT-6950IBC-1'
+    * def extItemBarcode2 = 'FAT-6950IBC-2'
+    * def extHoldingId = call uuid1
+    * def extInstanceId = call uuid1
+
+    # post a group and users
+    * def groupId = call uuid1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostGroup') { extUserGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId1), extUserBarcode: #(extUserBarcode1), extGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId2), extUserBarcode: #(extUserBarcode2), extGroupId: #(groupId) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostUser') { extUserId: #(extUserId3), extUserBarcode: #(extUserBarcode3), extGroupId: #(groupId) }
+
+    # post an instance
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostInstance') { extInstanceId: #(extInstanceId) }
+
+    # post a holding
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostHoldings') { extHoldingsRecordId: #(extHoldingId), extInstanceId: #(extInstanceId) }
+
+    # post a material type
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostMaterialType') { extMaterialTypeId: #(extMaterialTypeId), extMaterialTypeName: #(extMaterialTypeName) }
+
+    # post items
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(extItemId1), extItemBarcode: #(extItemBarcode1) }
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostItem') { extItemId: #(extItemId2), extItemBarcode: #(extItemBarcode2) }
+
+    # checkOut item1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostCheckOut') { extCheckOutUserBarcode: #(extUserBarcode1), extCheckOutItemBarcode: #(extItemBarcode1) }
+
+    # post page requests for user2
+    * def extRequestId1 = call uuid1
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostTitleLevelRequest') { requestId: #(extRequestId1), requesterId: #(extUserId2), extInstanceId: #(extInstanceId), extRequestLevel: "Title", extRequestType: "Page" }
```

Comment:
There is no support for parameter `extRequestLevel` in `@PostTitleLevelRequest`, request level is "Title" by default

Reply:
Thanks, removed